packages/gatsby-plugin-manifest/src/gatsby-node.js

```diff
@@ -12,7 +12,7 @@ sharp.simd(true)
 // Handle Sharp's concurrency based on the Gatsby CPU count
 // See: http://sharp.pixelplumbing.com/en/stable/api-utility/#concurrency
 // See: https://www.gatsbyjs.org/docs/multi-core-builds/
-sharp.concurrency(cpuCoreCount())
+sharp.concurrency(cpuCoreCount() - 1)
```

Comment:
Does concurrency make sense for the manifest plugin at all? It is for a single icon resizing, right?

Reply:
yeah — it wouldn't ever use more than one core

Reply:
Problem here is that sharp is a singleton so it will be shared with gatsby-plugin-sharp :'(

Reply:
But `gatsby-plugin-sharp` now has concurrency 1 (after this PR: https://github.com/gatsbyjs/gatsby/pull/28575):

https://github.com/gatsbyjs/gatsby/blob/f628c3801226f5b500d74b5b41bca4caa72a9e00/packages/gatsby-plugin-sharp/src/process-file.js#L19-L20

Reply:
I wonder what is the actual behavior currently if you have both plugins 🤔

Reply:
oh interesting :D I'll set it to one than. Basically last one in gatsby-config wins